### DISCUSSION TITLE
能创建一个APP出来吗？那样更方便一点

### DISCUSSION THREAD

**Discussion by xiae616** [dundunnp/hamibot-auto_xuexiqiangguo#497] [General]
基于虚拟机上面创建一个APP，51虚拟机能够安装学习强国，根据那个虚拟机的配置，基本所有手机都能适配

**Comment by dundunnp** [comment]
好的，我试试

**Reply by dundunnp** [reply]
> 好的，我试试

我试了大部分的安卓虚拟机都无法进行对战模型，请问你在51虚拟机上运行成功过脚本嘛，具体如何配置的？

**Reply by xiae616** [reply]
可以运行，vivo的手机

**Reply by xiae616** [reply]
> > 好的，我试试
> 
> 我试了大部分的安卓虚拟机都无法进行对战模型，请问你在51虚拟机上运行成功过脚本嘛，具体如何配置的？

不用什么配置，就一样的就没有修改过任何直接在虚拟机上运行，用的51虚拟机
